src/oas/service.ts

```diff
@@ -49,6 +49,7 @@ export const transformOasService: TranslateFunction<DeepPartial<OpenAPIObject> |
 
       ...toExternalDocs(document.externalDocs),
       extensions: getExtensions(document),
+      infoExtensions: getExtensions(document.info),
```

Comment:
Is the additional property `infoExtensions` needed because there's no node for `info` itself, so we can't distinguish between `x-widget` at the "top level" and `x-widget` nested under `info`?  

Looking past this ticket, are there any other [children of the top-level OpenAPI object](https://spec.openapis.org/oas/v3.0.1.html#fixed-fields) that would benefit from similar treatment, or even a more general solution?

Reply:
@EdVinyard  correct. a more general solution would likely require a larger rework of how extensions are stored

Reply:
Are there any other [children of the top-level OpenAPI object](https://spec.openapis.org/oas/v3.0.1.html#fixed-fields) that would benefit from similar treatment, or even a more general solution?  That is, should I write a follow-up ticket to fix other related errors that won't be fixed by this PR?